extensions/spring-config-server-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: "Quarkus Extension for reading properties from Spring Cloud Config Server"
```

Comment:
I have the intuition that this won't look good on code.quarkus.io.

Reply:
Indeed!

@jclingan any suggestions here?

Reply:
How about just `Spring Cloud Config Server`?

Reply:
Yeah, let's do that for now